Mathlib/Data/Set/Basic.lean

```diff
@@ -316,6 +316,8 @@ theorem mem_of_subset_of_mem {s₁ s₂ : Set α} {a : α} (h : s₁ ⊆ s₂) :
 theorem not_mem_subset (h : s ⊆ t) : a ∉ t → a ∉ s :=
   mt <| mem_of_subset_of_mem h
 
+alias not_mem_mono := not_mem_subset
```

Comment:
surely this is an anti lemma

Reply:
We have `Finset.not_mem_subset` and `Set.not_mem_mono`. What do you suggest the name of those two become?

Reply:
I'd say they should both be `not_mem_subset`